dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -1602,20 +1605,26 @@ private Response fireAction(final HttpServletRequest request,
     private void processPermissions(final FireActionForm fireActionForm,
                                     final ContentletDependencies.Builder formBuilder) {
 
-        if (UtilMethods.isSet(fireActionForm.getIndividualPermissions())) {
+        if (null != fireActionForm.getIndividualPermissions()) {
```

Comment:
did you implement this https://github.com/dotCMS/core/issues/22993#issuecomment-1248621401?

Reply:
yes